week-4/investigate/find.js

```diff
@@ -3,7 +3,7 @@ function find(str, char) {
 
   while (index < str.length) {
     if (str[index] === char) {
-      return index;
+      return index + 1;
```

Comment:
Be careful that small things like this don't creep into your pull request, there's no need for this to change 🙂